### ISSUE TITLE
[hipblas] rocblas.h instead of rocblas/rocblas.h

### ISSUE THREAD

**Issue by 7910f6ba7ee4** [rocm-arch/rocm-arch#800]
Attempting to install ROCm from the AUR without arch4edu and receiving a few build errors. Because the errors appear to not be linked, I'm creating two issues (#801). 

The first error is hopefully an easy fix, I'm receiving a build error for hipblas:
```
/home/whisper/.cache/paru/clone/hipblas/src/hipBLAS-rocm-5.1.3/library/src/hcc_detail/hipblas.cpp:7:10: fatal error: 'rocblas.h' file not found
#include "rocblas.h"
         ^~~~~~~~~~~
```

The [hipblas](https://github.com/ROCmSoftwarePlatform/hipBLAS/blob/develop/library/src/hcc_detail/hipblas.cpp) repository upstream appears to have this fixed, but the files from the AUR do not. 

In addition, is there a way to build a modified package and have it be accepted by the paru helper? Editing the file directly from the paru cache and executing `make install` in the build directory appears to work (build, install, and symlinks successful), but when attempting to install `rocm-hip-sdk`, I am still required to install hipblas. Apologies if this is a simple question.

**Comment by acxz** [comment]
> The [hipblas](https://github.com/ROCmSoftwarePlatform/hipBLAS/blob/develop/library/src/hcc_detail/hipblas.cpp) repository upstream appears to have this fixed

Here is the commit: https://github.com/ROCmSoftwarePlatform/hipBLAS/commit/84b593c7c6acf176b21841a0441801d5a254c62b

> a way to build a modified package and have it be accepted by the paru helper

`paru -Ui` inside the directory that has the `makepkg` should do the trick.

**Comment by tpkessler** [comment]
Patched with 4aee0cf588d440e660b9b285b5f4e5bb3e22c656

**Comment by tpkessler** [comment]
Patch merged upstream with 5.3.0
